types/koa-router/index.d.ts

```diff
@@ -2,7 +2,7 @@
 // Project: https://github.com/alexmingoia/koa-router/
 // Definitions by: Jerry Chin <https://github.com/hellopao>, Pavel Ivanov <https://github.com/schfkt>
 // Definitions: https://github.com/hellopao/DefinitelyTyped
-// TypeScript Version: 2.3
+// TypeScript Version: 2.8
```

Comment:
Can you keep this at 2.3? It lets people stuck on older versions of Typescript keep using this package.

Reply:
Sure thing, thanks for the review. I force-pushed a commit keeping the version at 2.3.